Library/Formula/docker-machine-nfs.rb

```diff
@@ -0,0 +1,14 @@
+class DockerMachineNfs < Formula
+  desc "Activates NFS on docker-machine"
+  homepage "https://github.com/adlogix/docker-machine-nfs"
+  url "https://github.com/adlogix/docker-machine-nfs/archive/0.2.0.tar.gz"
+  sha256 "ebc023657c044e706c8d9bc275d6b96de9d1f5178c7f6a90a8ce3bc167d1ec74"
+
+  def install
+    bin.install "docker-machine-nfs.sh" => "docker-machine-nfs"
+  end
+
+  test do
+    system "#{bin}/docker-machine-nfs"
```

Comment:
@bfontaine the script doesn't have a no-op mode, so any testing would involve making changes to the user's system.  would printing the help and checking to make sure that a particular string matches the output be preferable?